Examples/Tests/scraping/inputs_rz

```diff
@@ -41,7 +41,13 @@ electron.momentum_function_uy(x,y,z) = "if(x*x+y*y>0.0, -1.0*y/sqrt(x*x+y*y), 0.
 electron.momentum_function_uz(x,y,z) = "0"
 electron.do_not_deposit = 1
 
-diagnostics.diags_names = diag1
+diagnostics.diags_names = diag1 diag2
+
 diag1.intervals = 1
 diag1.diag_type = Full
 diag1.fields_to_plot = Ex
+
+diag2.intervals = 1
+diag2.diag_type = Full
+diag2.fields_to_plot = Er
+diag2.format = openpmd
```

Comment:
This includes openPMD output for an MR RZ case (max_level=1 in this input script), in order to have automated testing for the new bug fix.